src/main/webapp/stylesheets/teammatesCommon.css

```diff
@@ -310,6 +310,7 @@ Apply to thead to make headers not bold.
 
 .text-preserve-space {
     white-space: pre-wrap;
+    word-break: break-all;
```

Comment:
@rclakmal I think it is not an issue of inheritance/width etc.; using `break-all` is generally not ideal in any text that is meant to be read:
![text wrap](https://cloud.githubusercontent.com/assets/14101781/17122227/4223373a-530c-11e6-9bad-2f5428878f8a.png)

I think the problem lies somewhere with the line on `white-space: pre-wrap;`; maybe we shouldn't use it, maybe we should only we it when some condition is satisfied